src/Dockerfile

```diff
@@ -1,5 +1,6 @@
 # Start from latest GO base image
 FROM ubuntu:18.04
+LABEL org.opencontainers.image.source https://github.com/poseidonos/poseidonos-gui
```

Comment:
Is it possible to explain the purpose of this label?

Reply:
With this label, container images are linked to this repo (poseidonos-gui) registry and displayed.
If not,  container images are linked to poseidonos org.

Reply:
Thank you for the explanation.